pkg/dns/dns.go

```diff
@@ -61,41 +64,50 @@ func (c *Config) Validate() error {
 	return fmt.Errorf("invalid DNS provider %q", c.Provider)
 }
 
-// AskToConfigure reads the required DNS entries from a Terraform output,
-// asks the user to configure them and checks if the configuration is correct.
-func (c *Config) AskToConfigure(ex *terraform.Executor) error {
-	dnsEntries, err := readDNSEntries(ex)
-	if err != nil {
-		return err
-	}
+// ManualConfigPrompt returns a terraform.ExecutionHook which prompts the user to configure DNS
+// entries manually and verifies the entries were created successfully.
+func ManualConfigPrompt(c *Config) terraform.ExecutionHook {
+	return func(ex *terraform.Executor) error {
+		dnsEntries, err := readDNSEntries(ex)
+		if err != nil {
+			return err
+		}
 
-	fmt.Printf("Please configure the following DNS entries at the DNS provider which hosts %q:\n", c.Zone)
-	prettyPrintDNSEntries(dnsEntries)
+		fmt.Printf("Please configure the following DNS entries at the DNS provider which hosts %q:\n", c.Zone)
```

Comment:
Perhaps we could have another unexported function to ask questions, which would take dnsEntries as an argument? That would simplify the testing.

Reply:
What do you mean by a function for asking questions? Wrap `fmt.Printf()` in a function?
Anyway, IMO this belongs in a "refactor `pkg/dns`" task, i.e. out of scope.

Reply:
I meant, instead of taking executor as an argument and then execute the prints etc. it would be better to have a function like this, which takes an executor, gets dnsEntries using helper and passes that to function which does prints and reads user input. This way, we could fully test prints and reading user input.

Reply:
OK, feel free to open an issue if you think we should do that. I'm resolving this since I think we agree this is out of scope here.

Reply:
Not sure if such small things are worth creating an issue and allocating dedicated time. Seems more like a thing to do together with something IMO.